openlibrary/templates/borrow.html

```diff
@@ -127,12 +134,7 @@ <h2 class="sansserif">This title is not available to borrow.</h2>
                 <p>Look for something else in the <a href="/borrow">lending library</a>?</p>
             </div>
     $elif not page.ocaid:
```

Comment:
There used to be Overdrive code in the `elif`, now deprecated. However removing the conditional would mean that the subsequent condition's code could be executed if there is no `page.ocaid`. Because this nested-if is so convoluted, and I don't have the time or people available to fully explain it, it seemed safest to just cause a `pass` on the condition.